content/get-haiku/r1beta4/release-notes.md

```diff
@@ -0,0 +1,189 @@
++++
+type = "article"
+title = "R1/beta4 – Release Notes"
+tags = []
+date = "2020-12-23 00:00:00Z"
+draft = "false"
++++
+
+The fourth beta for Haiku R1 over a year and a half of hard work to improve Haiku's hardware support and its overall stability, and to make lots more software ports available for use. Over 400 bugs and enhancement tickets have been resolved for this release.
+
+Please keep in mind that this is beta-quality software, which means it is feature complete but still contains known and unknown bugs. While we are increasingly confident in its stability, we cannot provide assurances against data loss.
+
+For most of this release cycle, waddlesplash was [employed as a contractor](https://www.haiku-os.org/news/2021-08-25_hiring_waddlesplash/) to work on Haiku. His contract is presently ongoing, supported by the generous donations of readers like you to Haiku, Inc., a 501(c)3 non-profit.
+
+*To download Haiku or learn how to upgrade from R1/beta3, see "[Get Haiku!](/get-haiku/)". For press inquiries, see "[Press contact](#press-contact)".*
+
```

Comment:
This should include a link to the Trac milestone with the complete list of tickets for people who just want to read about everything